backend/src/config.js

```diff
@@ -18,12 +18,21 @@ const config = {
     domain: process.env.SESSION_DOMAIN,
   },
   logFormat: '[:date[iso]] :method :url :status :response-time ms - :req[header]',
+  aws: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID || 'FAKE',
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'ALSO FAKE',
+    kinesisEndpoint: 'http://localhost:4567',
+  },
+  eventAuthToken: process.env.EVENT_AUTH_TOKEN || 'secret',
 };
 
 if (process.env.NODE_ENV === 'production') {
   config.session.proxy = true;
   config.session.secureCookie = true;
 
+  // Let the SDK derive the endpoint from the region
+  config.aws.kinesisEndpoint = undefined;
```

Comment:
I'm not sure I like this pattern of sneaky const override. I think I'd rename to base config and then return something like a spread of base + env specific config.